### PR TITLE
Reducing LSP server messages on startup.

### PR DESCRIPTION
**Pull request by sdedic**
* The debugging messages suppressed by default, added `Verbose Console` option that enables it back.
* Merged the outputs `Java` and `Java Language Server` - OK, @JaroslavTulach, @dbalek ?
* Start/ready messages short-displayed in the status bar

Will mark with approriate milestone & ask Coordinator for approval after this is functionally approved first.

### CODE REVIEW

File: java/java.lsp.server/vscode/src/extension.ts [hunk 1]
```diff
@@ -205,7 +207,7 @@ export function activate(context: ExtensionContext) {
 
         let debugDescriptionFactory = new NetBeansDebugAdapterDescriptionFactory();
         context.subscriptions.push(vscode.debug.registerDebugAdapterDescriptorFactory('java-polyglot', debugDescriptionFactory));
-        window.showInformationMessage('Java Polyglot Debug Adapter ready.');
+        window.setStatusBarMessage('Java Polyglot Debug Adapter ready.', 5000);
```
Comment (JaroslavTulach) [inline java/java.lsp.server/vscode/src/extension.ts:210]: We don't really need this message at all. It is a heritage from the early attempts to make the Debug Adapter Protocol work.

File: java/java.lsp.server/vscode/src/extension.ts [hunk 2]
```diff
@@ -85,9 +88,8 @@ export function activate(context: ExtensionContext) {
         }
     });
 
-    let log = vscode.window.createOutputChannel("Java Language Server");
     log.appendLine("Launching Java Language Server");
-    vscode.window.showInformationMessage("Launching Java Language Server");
+    vscode.window.setStatusBarMessage("Launching Java Language Server", 2000);
```
Comment (JaroslavTulach) [inline java/java.lsp.server/vscode/src/extension.ts:92]: Great. I wasn't aware of this method. Probably added in newer version of VSCode.

File: java/java.lsp.server/vscode/package.json [hunk 3]
```diff
@@ -35,7 +35,12 @@
 					],
 					"default": null,
 					"description": "Specifies the JDK on which the Java language server should be run"
-				}
+				},
+                                "netbeans.verboseConsole": {
```
Comment (JaroslavTulach) [inline java/java.lsp.server/vscode/package.json:39]: Let's do `netbeans.verbose` only or `netbeans.loglevel`, or `netbeans.debug`. CCing @jisedlac

File: java/java.lsp.server/vscode/package.json [hunk 4]
```diff
@@ -35,7 +35,12 @@
...
+                                        "type": "boolean",
+                                        "default": false,
+                                        "description": "Enables verbose messages from the Java Lanuage server on the console"
```
Comment (JaroslavTulach) [inline java/java.lsp.server/vscode/package.json:42]: Re. "on the console" - I don't think this makes sense from the point of view of VSCode user. The user sees no "console" as far as I can say. "Enables debug messages from the Java Language Server" (in the output tab).

Btw. typo "Lanuage".

**Review by JaroslavTulach — APPROVED** [review]
Fix the typo, option description. Consider renaming the option.

As a continuation of this effort, I'd like the [AbstractLspStatusDisplayer](https://github.com/apache/netbeans/blob/master/java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/ui/AbstractLspStatusDisplayer.java) to also display its messages via `vscode.window.setStatusBarMessage`. That can however be done in a subsequent PR.

**Review by lkishalmi — CHANGES_REQUESTED** [review]
I guess @JaroslavTulach's approve means he is generally fine with the PR, but please fix the mentioned typo, and other small things.

**Review by lkishalmi — APPROVED** [review]
Approved.

**Comment by lkishalmi** [comment]
Well, I guess there are some conflict wioth your lates t work. Please resolve them.

**Comment by sdedic** [comment]
rebased on latest `delivery`; squashed.

**Pull request merged**